Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridDataConnection.cs

```diff
@@ -585,6 +604,13 @@ public int IndexOf(object dataItem)
             return -1;
         }
 
+        public void LoadMoreItems(uint count)
+        {
+            Debug.Assert(_loadingOperation == null, "Expected _loadingOperation == null.");
+            _loadingOperation = _incrementalItemsSource.LoadMoreItemsAsync(count);
+            _loadingOperation.Completed = OnLoadingOperationCompleted;
```

Comment:
I think we need to be careful here and handle the case where the customer's LoadMoreItemsAsync returns null:

```
if (_loadingOperation != null)
{
    _loadingOperation.Completed = OnLoadingOperationCompleted; 
}
```

Reply:
Good point